sympy/stats/joint_rv_types.py

```diff
@@ -394,6 +395,159 @@ def MultivariateEwens(syms, n, theta):
+    distribution.
+
+    The joint pdf can be found at [1].
+
+    Parameters
+    ==========
+
+    syms: list/tuple/set of symbols for identifying each component
+    delta: A constant in range [0, 1]
+    v: positive real
+    lamda: a list of positive reals
+    mu: a list of positive reals
+
+    Returns
+    =======
+
+    A Random Symbol
+
+    Examples
+    ========
+
+    >>> from sympy.stats import density
+    >>> from sympy.stats.joint_rv import marginal_distribution
+    >>> from sympy.stats.joint_rv_types import GeneralizedMultivariateLogGamma
+    >>> from sympy import symbols
+    >>> v = 1
+    >>> l, mu = [1, 1, 1], [1, 1, 1]
+    >>> d = 0.5
+    >>> y = symbols('y_1:4', positive=True)
+    >>> Gd = GeneralizedMultivariateLogGamma('G', d, v, l, mu)
+    >>> density(Gd)(y[0], y[1], y[2])
+    0.5*Sum(0.5**n*exp((n + 1)*(y_1 + y_2 + y_3) - exp(y_1) - exp(y_2) -
+    exp(y_3))/gamma(n + 1)**3, (n, 0, oo))
+
+    References
+    ==========
+
+    .. [1] https://en.wikipedia.org/wiki/Generalized_multivariate_log-gamma_distribution
+    .. [2] https://www.researchgate.net/publication/234137346_On_a_multivariate_log-gamma_distribution_and_the_use_of_the_distribution_in_the_Bayesian_analysis
+
+    Note
+    ====
+    If the GeneralizedMultivariateLogGamma is too long to type use,
+    `from sympy.stats.joint_rv_types import GeneralizedMultivariateLogGamma as GMVLG`
+    If you want to pass the matrix omega instead of the constant delta, then use,
+    GeneralizedMultivariateLogGammaOmega.
+
+    """
+    return multivariate_rv(GeneralizedMultivariateLogGammaDistribution,
+                            syms, delta, v, lamda, mu)
+
+def GeneralizedMultivariateLogGammaOmega(syms, omega, v, lamda, mu):
+    """
+    Extends GeneralizedMultivariateLogGamma.
+
+    Parameters
+    ==========
+
+    syms: list/tuple/set of symbols for identifying each component
+    omega: A square matrix
+           Every element of square matrix must be absolute value of
+           sqaure root of correlation coefficient
+    v: positive real
+    lamda: a list of positive reals
+    mu: a list of positive reals
+
+    Returns
+    =======
+
+    A Random Symbol
+
+    Examples
+    ========
+
+    >>> from sympy.stats import density
+    >>> from sympy.stats.joint_rv import marginal_distribution
+    >>> from sympy.stats.joint_rv_types import GeneralizedMultivariateLogGammaOmega
+    >>> from sympy import Matrix, symbols
+    >>> omega = Matrix([[1, 0.5, 0.5], [0.5, 1, 0.5], [0.5, 0.5, 1]])
+    >>> v = 1
+    >>> l, mu = [1, 1, 1], [1, 1, 1]
+    >>> G = GeneralizedMultivariateLogGammaOmega('G', omega, v, l, mu)
+    >>> y = symbols('y_1:4', positive=True)
+    >>> density(G)(y[0], y[1], y[2])
+    sqrt(2)*Sum((1 - sqrt(2)/2)**n*exp((n + 1)*(y_1 + y_2 + y_3) - exp(y_1) -
+    exp(y_2) - exp(y_3))/gamma(n + 1)**3, (n, 0, oo))/2
+
+    References
+    ==========
+
+    See refernces of GeneralizedMultivariateLogGamma.
+
+    Note
```

Comment:
I think this needs to be Notes (plural) and a blank line left after the underline...but I could be wrong,